hash_assertion_test.py

```diff
@@ -0,0 +1,18 @@
+
+from cgi import test
+
+
+def hash_test(test_input: str) ->str:
+    hash_value: int = hash(test_input)
+    hex_str: str = hex(hash_value)
+    return hex_str 
+
+def hash_assertion():
+    a = 'test'
+    b = 'test'
+    print(hash_test(a))
+    print(hash_test(b))
+    assert(hash_test(a) == hash_test(b))
+
+
+print(hash_assertion())
```

Comment:
This `print()` call doesn't do anything meaningful

`hash_assertion()` does not return a value
which means it'll print the same thing as `print(None)`

this line should instead just be `hash_assertion()`, because you still want to call the function, it's just that there's nothing to print